radixdlt/src/main/java/com/radixdlt/CerberusModule.java

```diff
@@ -100,14 +106,22 @@ private EpochRx epochRx(
 	private ValidatorSet validatorSet(
 		@Named("self") ECKeyPair selfKey
 	) {
-		return ValidatorSet.from(Collections.singleton(Validator.from(selfKey.getPublicKey(), UInt256.ONE)));
+		return ValidatorSet.from(Collections.singleton(Validator.from(selfKey.getPublicKey(), UInt128.ONE)));
 	}
 
 	@Provides
 	@Singleton
-	private PacemakerImpl pacemaker() {
+	private ScheduledTimeoutSender timeoutSender() {
+		return new ScheduledTimeoutSender(Executors.newSingleThreadScheduledExecutor());
+	}
+
+	@Provides
+	@Singleton
+	private FixedTimeoutPacemaker pacemaker(

```

Comment:
Return type should be `Pacemaker`, rather than the implementation `FixedTimeoutPacemaker`.